README.md

```diff
@@ -39,8 +39,7 @@ Apache NetBeans is an open source development environment, tooling platform, and
 
   * Git
   * Ant 1.9.9 or above
-  * JDK 11 (to build NetBeans)
-  * JDK 11 or above (to run NetBeans)
+  * JDK 11 or above (to build and run NetBeans)
   * MinGW (optional), to build Windows Launchers
```

Comment:
This might deserve also a note about profiler binaries...

Reply:
I don't know the requirements for the profiler build. I could add a bullet point if its simple enough. If it's requirements are more complex maybe it would deserve its own readme?

Reply:
Yes, maybe a separate section for native binaries, and probably needs rewriting given - https://github.com/apache/netbeans-native-launchers cc/ @ebarboni

Reply:
@ebarboni let's take the Swift discussion on list once NB14 out.  I've got some infra I can test on at least.  (And have another variant of the Swift launcher that works for IDE and RCP in NBPackage).  @oyarzun probably good to get in on it.

Reply:
awesome, my understanding (taken from other comments and here) is that this is good to go, the profiler/installers(?) section will be added via a follow up PR.